packages/runtime/src/createGatewayRuntime.ts

```diff
@@ -991,6 +992,16 @@ export function createGatewayRuntime<
     useRetryOnSchemaReload({ logger }),
   ];
 
+  if (config.subgraphErrors !== false) {
+    basePlugins.push(
+      useSubgraphErrorPlugin(
+        typeof config.subgraphErrors === 'object'
+          ? config.subgraphErrors
+          : undefined,
+      ),
+    );
+  }
+
```

Comment:
Isn't this breaking? You now have to explicitly enable `subgraphErrors` in the config.

Reply:
Yes it is enabled by default already. I just make it configurable with this PR.

Reply:
Understood.